bundles/edu.kit.ipd.sdq.pcm.simulation.scheduler.exact/src/edu/kit/ipd/sdq/pcm/simulation/scheduler/exact/ExactSchedulingFactory.java

```diff
@@ -85,8 +86,8 @@ public ExactSchedulingFactory(final SchedulerModel model, final String scheduler
     private final Map<String, PriorityManagerImpl> manager_map = new Hashtable<String, PriorityManagerImpl>();
 
     /* Loads scheduler configuration */
-    public IActiveResource getResource(final SchedulerModel model,
-            final String schedulerName, final long numReplicas, final String sensorDescription) {
+    public IActiveResource getResource(final SchedulerModel model, final String schedulerName, final long numReplicas
+            , final String sensorDescription, IResourceTableManager resourceTableManager) {

```

Comment:
The line break looks missplaced. I'd expect the comma to be on the previous line.

Reply:
worked in review finding; reformatted file using Palladio code formatter